labelbox/data/serialization/labelbox_v1/objects.py

```diff
@@ -42,6 +46,52 @@ def validate_subclasses(cls, value, field):
         return value
 
 
+class _Coordinates(BaseModel):
+    """Union of the possible coordinate lists in tiled imagery exports"""
+    coordinates: Union[List[float], List[List[float]], List[List[List[float]]]]
```

Comment:
Instead of creating a class that is the union of all other potential coordinate types, I would just type each geometry field individually.